### PR TITLE
Add alot of things

### PR DESCRIPTION
**Pull request by ddvleeuwen**
- Do linting
- Remove `ThemeSelector` from header, only keep option on mobile menu.
- Make login page
- Add buttonRoute (login) to header
- Add Team page
- Make Crazy88 page
- Make Empty Bingo page
- Add vercel `/api` routing
- Add local `/api` routing

### CODE REVIEW

File: src/components/crazy/CrazyChallenge.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  const [ percentage, setPercentage ] = useState(0);
+  const [ uploading, setUploading ] = useState(false);
+  const [ error, setError ] = useState<string | undefined>(undefined);
+
+  const uploadFile = (e: FormEvent, challenge: Challenge) => {
+    const formData = new FormData((e.target as HTMLElement).parentElement as HTMLFormElement);
+    setUploading(true);
+
+    uploadChallenge(challenge, formData, setPercentage).then((response) => {
+      console.log(response.data);
```
Comment (ddvleeuwen) [inline src/components/crazy/CrazyChallenge.tsx]: I should probably remove this `console.log`

**Comment by kaasbroodju** [comment]
Bij het uploaden kan ik nog alles uploaden, kan je het beperken tot alleen video en foto?

**Comment by kaasbroodju** [comment]
When a submission is done, the upload button is still shown. In the backend after a submission is approved it cannot be changed. Could you remove the upload button, when the submission is approved?

**Comment by kaasbroodju** [comment]
The border of the upload button is different when the upload failed from the submission denies. Please check again for consistency.

**Comment by kaasbroodju** [comment]
The join code is always 6 characters long, could you reflect that in the placeholder text on the login page

**Comment by kaasbroodju** [comment]
When unauthed navigating from the home screen pressing _Let's play!_ I get redirected to the login page, after that when I use the back button of my browser I keep getting redirected to the login page instead of the home page.

**Comment by kaasbroodju** [comment]
When logged in I still see the login button, shouldn't that be a sign-out button?

**Comment by kaasbroodju** [comment]
When a submission is in review, it cannot be changed. When uploading this will always result into an error.

**Comment by ddvleeuwen** [comment]
> The border of the upload button is different when the upload failed from the submission denies. Please check again for consistency.

I'm not 100% sure what you mean by this, I think you mean the borders are different so the button and error message fit together nicely. Not sure how you'd want it to look otherwise.

**Comment by kaasbroodju** [comment]
> When unauthed navigating from the home screen pressing _Let's play!_ I get redirected to the login page, after that when I use the back button of my browser I keep getting redirected to the login page instead of the home page.

This is still not fixed

**Comment by kaasbroodju** [comment]
Branch can be merged if back button issue is resolved.

**Pull request merged**